crates/bestool/src/actions/tamanu/alerts.rs

```diff
@@ -460,6 +519,49 @@ async fn execute_alert(
 					.into_diagnostic()
 					.wrap_err("sending email")?;
 			}
+			SendTarget::Zendesk {
+				endpoint,
+				method,
+				ticket_form_id,
+				custom_fields,
+				..
+			} => {
+				if dry_run {
+					println!("-------------------------------");
+					println!("Alert: {}", alert.file.display());
+					println!("Endpoint: {}", endpoint);
+					println!("Subject: {subject}");
+					println!("Body: {body}");
+					continue;
+				}
+
+				let req = json!({
+					"request": {
+						"subject": subject,
+						"ticket_form_id": ticket_form_id,
+						"custom_fields": custom_fields,
+						"comment": { "html_body": body },
+						"requester": requester.map(|r| json!({ "name": r }))
+					}
+				});
+
+				let mut req_builder = reqwest::Client::new().post(endpoint.clone()).json(&req);
```

Comment:
Can we initialise the client outside once of the loop?

Reply:
Done (I hope)